### ISSUE TITLE
Typescript types not available in setup with Yarn PnP, Yarn VS Code SDK

### ISSUE THREAD

**Issue by marvinruder** [jimp-dev/jimp#1192]
# Expected Behavior

Yarn types are available in IDE and used by Typescript for type-checking.

# Current Behavior

Types are not available, IDE shows `(alias) type Jimp = any`. Typescript does not throw an error at `Jimp.thisFunctionDoesNotExist();`.

# Failure Information (for bugs)

## Steps to Reproduce

1. `git clone git@github.com:marvinruder/jimp-no-types-available.git && cd jimp-no-types-available`
2. `yarn`
3. `yarn build`
4. `yarn start`
5. When using VS Code, hovering over `import Jimp from "jimp";` shows `(alias) type Jimp = any`

**Screenshots**

<img width="364" alt="image" src="https://user-images.githubusercontent.com/18495294/217214529-4c7ddea4-e5ba-4bd8-88a4-2d2c88d73794.png">

## Context

Context is given in [marvinruder/jimp-no-types-available](https://github.com/marvinruder/jimp-no-types-available) and was briefly mentioned in #1189.

- Jimp Version: 0.22.4
- Operating System: macOS 13.2 (build 22D49)
- Node version: 19.6.0

**Comment by marvinruder** [comment]
Already tried `yarn unplug jimp`, has no effect, `type Jimp` is still resolved to `any`. 

In `tsconfig.json`, `"skipLibCheck": true` appears to be required, removing it throws 27 errors: `error TS2307: Cannot find module '@jimp/core' or its corresponding type declarations.`

**Comment by MikuZZZ** [comment]
Add these lines to `.yarnrc.yml` should solve this problem

```
packageExtensions:
  jimp@*:
    dependencies:
      '@jimp/core': '*'
```

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v1.0.2`](https://github.com/jimp-dev/jimp/releases/tag/v1.0.2) :rocket:
